test/e2e/installplan_e2e_test.go

```diff
@@ -129,6 +129,10 @@ func TestCreateInstallPlanManualApproval(t *testing.T) {
 		},
 	}
 
+	// Attempt to get the catalog source before creating install plan
+	_, err = fetchCatalogSource(t, crc, ocsConfigMap, testNamespace, catalogSourceSynced)
```

Comment:
after these changes, the test installplan didn't have a `source` and `sourceNamespace` in the spec anymore

Reply:
It looks like the `source` and `sourceNamespace` are not in [master](https://github.com/operator-framework/operator-lifecycle-manager/blob/207752168e6c798dd252318d8079d6b38ea5ff44/test/e2e/installplan_e2e_test.go#L125) either.